test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -13,18 +13,15 @@ fixture('English Leaks');
 const ignoredMocks = [
   'success-with-interaction-code.json',
   'identify-with-third-party-idps.json',
-  'identify-with-no-sso-extension.json',
-  'identify-with-device-probing-loopback.json',
-  'identify-with-device-probing-loopback-3.json',
   'identify-with-apple-redirect-sso-extension.json', // flaky on bacon
   'identify-unknown-user.json',
   'error-user-is-not-assigned.json',
   'error-internal-server-error.json',
   'error-forgot-password.json',
   'authenticator-verification-select-authenticator.json',
-  'authenticator-verification-okta-verify-signed-nonce-loopback.json',
   'error-with-failure-redirect.json',
-  'identify-recovery-with-recaptcha-v2.json'
+  'identify-recovery-with-recaptcha-v2.json',
+  'consent-enduser.json'
```

Comment:
Is there a reason to add this mock to ignore-list? If it's testcafe failures on bacon, please retrigger another run. This file needs to go through enleaks test.

cc: @nikhilvenkatraman-okta

Reply:
+ 1 to what Ani said, why are we adding these mocks back. Can you describe what the new failures are ?

Reply:
@sherizada-okta made a recent fix to this https://github.com/okta/okta-signin-widget/pull/1977/files. Need to check why we are still seeing leaks on this screen